pages/careers/_open-positions.html

```diff
@@ -237,5 +320,267 @@ <h4>Help us a create a fundamentally better DevOps experience!</h4>
+        <strong>Work alongside customers.</strong>
+        You'll work directly with our customers to understand what problems they are facing, design solutions to
+        those problems, guide the team in user testing those solutions, gather feedback, and iterate.
+      </li>
+      <li>
+        <strong>Drive our design practices.</strong>
+        Help improve how we design products as a company by streamlining all of our design processes, including
+        designs prints, user testing, user story mapping, wireframing, prototyping, customer communication, and more.
+      </li>
+    </ul>
+    <h4>Your Ideal Background</h4>
+    <ul>
+      <li>
+        <strong>Product sensibility.</strong>
+        We're looking for a product designer—someone who can be an equal partner in helping us design and develop our
+        products. We'll look to you for leadership, insights, and methodologies that will help us translate our shared
+        vision into a fantastic product. Experience with launching products from 0 to 1 is a plus.
+      </li>
+      <li>
+        <strong>Design skill.</strong>
+        At the end of the day, the designs you create should look great and be easy to use. Help us make Gruntwork
+        customers smile. :)
+      </li>
+      <li>
+        <strong>Domain expertise.</strong>
+        We don't expect you to know everything coming in, but to design products in our space, you must be able to
+        quickly build a deep understanding of our domain: DevOps, CLI tools, SaaS software, CI / CD, and infrastructure
+        as code. If you happen to have prior experience in software engineering, working on dev tools, or other
+        technical domains, that is a big plus.
+      </li>
+      <li>
+        <strong>Bonus: HTML/CSS.</strong>
+        If you can occasionally roll up your sleeves and work directly in HTML/CSS, you can help us make sure the final
+        product is true to your vision.
+      </li>
+      <li>
+        <strong>Values.</strong>
+        You are inspired by <a href="/about/#our-values">our values</a>. Bonus points for a sense of humor, empathy,
+        and curiosity.
+      </li>
+    </ul>
+    <h4>Compatible time zones</h4>
+    <p>
+      You'll be working with a team in the <strong>European time zones</strong>, so you can be located in almost any
+      country as long as your time zone is no further west than the UK (GMT+0/BST) and no further east than Germany
+      (GMT+2). We've found that when everyone on the team is located in similar time zones, it's easier to collaborate
+      and there's much less pressure to stay up late or get up early, so this is a hard constraint, even if you're
+      willing to work hours different from your current time zone.
+    </p>
+    <h4>Help us a create a fundamentally better DevOps experience!</h4>
+    <p>
+      If you're excited by the above:<br />
+      <!-- TODO: Fill in the proper link!!! -->
+      <a
+        href="#TODO:fill-me-in!"
```

Comment:
This exists in Workable as well.

Reply:
Ah, yea, I see all of the jobs now. I filled out all of their descriptions based on this PR and published them on our careers page.